From 8064523eb18be802079565fe5deeb72c9149f2eb Mon Sep 17 00:00:00 2001
From: Maarten Duijn <maarten@tulipsolutions.nl>
Date: Wed, 13 Mar 2019 11:48:57 +0100
Subject: [PATCH 1/2] Set core:internal visibility to public

This is required because rules_kotlin's dependencies work different from Java's
A Kotlin rule might need to explicitly name the dependency below.
See https://github.com/grpc/grpc-java/issues/4258
---
 core/BUILD.bazel | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git core/BUILD.bazel core/BUILD.bazel
index 61fa3bbf8..60a2e9928 100644
--- core/BUILD.bazel
+++ core/BUILD.bazel
@@ -37,7 +37,7 @@ java_library(
     resources = glob([
         "src/bazel-internal/resources/**",
     ]),
-    visibility = ["//:__subpackages__"],
+    visibility = ["//visibility:public"],
     deps = [
         ":core",
         "//context",

base-commit: 206a2e39333e991963adc7da74cb05dd2410c1a5
-- 
2.21.0

